### ISSUE TITLE
Scrolling is a bit choppy.

### ISSUE THREAD

**Issue by eth-p** [Riey/rust-pager#6]
I did manage to get `rp` working with process substitution, and it's a pretty nice pager without any of the bulk that comes with `less`. Scrolling is a bit choppy and slow with iTerm2 though, unfortunately:
 
https://user-images.githubusercontent.com/32112321/112741008-983be480-8f36-11eb-9e3b-3e0b17939662.mov

**Comment by Riey** [comment]
Can you try this branch? `cargo install --git https://github.com/Riey/rust-pager --branch reduce-flicker --force`

**Comment by eth-p** [comment]
The flickering is completely gone, but the input is still a bit slow I think. If I scroll down on the trackpad, it takes a few seconds for the pager to catch up to where I scrolled:

https://user-images.githubusercontent.com/32112321/112744623-908c3800-8f56-11eb-8d1d-155d2c0b50b4.mov

It's especially slow once it reaches the bottom. I think what happens is that `rp` reaches the bottom, it keeps trying to scroll down. Before the flicker was fixed, I noticed it constantly redrawing even though there was nothing left to scroll.
